### PR TITLE
Sms campaign pending fix

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          | ` feat/sms-campaign-pending`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-5201
| License          | BSD 3-Clause

## Description

Add dependencie for sms app (Modal) because micro-front need to work at standalone.

### CODE REVIEW

File: packages/manager/apps/sms/package.json [hunk 1]
```diff
@@ -63,4 +64,4 @@
     "ovh-manager-webfont": "^1.2.0",
     "webpack-merge": "^4.2.2"
   }
-}
\ No newline at end of file
+}
```
Comment (marie-j) [inline packages/manager/apps/sms/package.json:68]: No need to add `angular-ui-bootsrap` too ?

**Review by marie-j — COMMENTED** [review]
Overall, changes look good to me. However we tend to follow Angular's commit guidelines (https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) 
May I suggest to use a verb as the purpose is to describe an action made 
**suggestion**: `fix(deps): add missing dependency for modal`
Moreover the `why` reason should be in the body not in the header

File: packages/manager/apps/sms/package.json (hunk 1 above)
Comment (Heiiji) [reply packages/manager/apps/sms/package.json:68]: done on last commit, my bad :)

File: packages/manager/modules/sms/package.json [hunk 2]
```diff
@@ -32,10 +32,12 @@
     "@ovh-ux/ng-ovh-telecom-universe-components": "^5.0.0 || ^6.0.0",
     "@ovh-ux/ng-ovh-ui-confirm-modal": "^2.0.0",
     "@ovh-ux/ng-pagination-front": "^10.0.0",
+    "@ovh-ux/ng-ui-router-layout": "^4.0.0",
     "@ovh-ux/ui-kit": "^4.1.12",
     "@uirouter/angularjs": "^1.0.15",
     "angular": "^1.7.5",
     "angular-messages": "^1.7.6",
+    "angular-ui-bootstrap": "~1.3.3",
```
Comment (cbourgois) [inline packages/manager/modules/sms/package.json:40]: Not really a peer deps for the `@ovh-ux/manager-sms` module (not imported), but it's a peer deps of one peer dep: `@ovh-ux/ng-ui-router-layout`.

Comment (Heiiji) [reply packages/manager/modules/sms/package.json]: thanks for explanations about AngularJS things ! i've done on last commit :)

**Review by Jisay — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Pull request merged**